### ISSUE TITLE
Only CCTV1 (semi-)works

### ISSUE THREAD

**Issue by RunasSudo** [RunasSudo/plugin.video.cntv-live#17]
- [x] Servers listed in `live.do` for other channels do not respond. (Fixed.)

- [X] CCTV1 connection stops after a while. (Fixed.)

**Comment by RunasSudo** [comment]
cctv2 -> 60.210.10.12

**Comment by RunasSudo** [comment]
All channels are currently broken.
```
cctv1.vtime.cntv.cloudcdn.net -> 60.210.10.105 (FAIL)
cctv1.vtime.cntv.cloudcdn.net -> 183.56.172.32 (FAIL)
cctv1.vtime.cntv.cloudcdn.net -> 183.56.172.33 (FAIL)
cctv1.vtime.cntv.cloudcdn.net -> 183.56.172.34 (FAIL)
cctv1.vtime.cntv.cloudcdn.net -> 183.56.172.35 (FAIL)
cctv1.vtime.cntv.cloudcdn.net -> 60.210.10.103 (FAIL)
cctv1.vtime.cntv.cloudcdn.net -> 60.210.10.104 (FAIL)
cctv2.vtime.cntv.cloudcdn.net -> 113.107.250.34 (FAIL)
cctv2.vtime.cntv.cloudcdn.net -> 113.107.250.35 (FAIL)
cctv2.vtime.cntv.cloudcdn.net -> 113.107.250.36 (FAIL)
cctv2.vtime.cntv.cloudcdn.net -> 113.107.250.37 (FAIL)
```
(etc.)

```[runassudo@NEXUS tools]$ curl "60.210.10.105"
curl: (18) transfer closed with 28 bytes remaining to read```

**Comment by RunasSudo** [comment]
`hds2` on CCTV1 returns a valid FLV file stream (without `Content-Type` header).

**Comment by RunasSudo** [comment]
Closed in 8c96620f9d195a81659c79c85983705095d86c37
